cucoslib/utils.py

```diff
@@ -608,5 +610,13 @@ def case_sensitivity_transform(ecosystem, name):
 
     return name
 
+def get_session_retry(retries=3, backoff_factor=0.2, status_forcelist=(404, 500, 502, 504), session=None):
```

Comment:
I'd probably trade 404 for 503 here, but I don't know how the list was worked out :)